doc/ext/traffic-server.py

```diff
@@ -389,7 +389,7 @@ def get_objects(self):
                                       'rev-parse', '--abbrev-ref', 'HEAD'])
 
 
-def make_github_link(name, rawtext, text, lineno, inliner, options={}, content=[]):
+def make_github_link(name, rawtext, text, lineno, inliner, options=None, content=None):
```

Comment:
Wow, what a crazy design.
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/